dll/directx/wine/d3dcompiler_43/reflection.c

```diff
@@ -1107,13 +1107,13 @@ static HRESULT d3dcompiler_parse_stat(struct d3dcompiler_shader_reflection *r, c
 
     skip_dword_unknown(&ptr, 1);
 
-    read_dword(&ptr, &r->input_primitive);
+    read_dword(&ptr, (DWORD*)&r->input_primitive);
```

Comment:
These aren't 64-bit-specific, are they? (also, this is Wine code, so perhaps disable the warning instead)
Same for windowscodecs.